### PR TITLE
Intendo issue91

### PR DESCRIPTION
**Pull request by intendo**
Made grammar more correct to find ATOM and HETOM.
Fixed documentation.
Added tests.

### CODE REVIEW

**Review by aaronrphillips — APPROVED** [review]

File: apbs/chemistry/atom.py [hunk 1]
```diff
@@ -20,39 +20,27 @@ def __init__(self, *args, **kwargs):
         """
         Arguments:
 
-        :param id: A unique identifier for this Atom
-        :type id: int
-        :param field_name: Specifies the type of PQR entry and should either
+        :param int id: A unique identifier for this Atom
```
Comment (sobolevnrm) [inline apbs/chemistry/atom.py:23]: What versions of Sphinx support this combined type syntax?  Are they compatible with readthedocs?

File: apbs/test/test_chemistry/test_atom_list.py [hunk 2]
```diff
@@ -1,76 +1,73 @@
...
+    """Return the AtomList from the file generated."""
 
-    fp = tmp_path / "atom.pdb"
+    fqpn = tmp_path / "atom.pdb"
 
-    with open(fp, "w") as f:
+    with open(fqpn, "w") as f:
         f.write(
-            "HETATM    1  C    ALK    1       "
+            "HETOM    1  C    ALK    1       "
```
Comment (sobolevnrm) [inline apbs/test/test_chemistry/test_atom_list.py:15]: "HETOM" is not a PDB keyword.  "HETATM" is correct.

Where did this change come from?  What are you testing against?  This is the best reference:  https://www.wwpdb.org/documentation/file-format-content/format33/v3.3.html

File: apbs/test/test_chemistry/test_atom_list.py [hunk 3]
```diff
@@ -1,76 +1,73 @@
...
             "1.000   4.000   7.000 0.000 1.000\n"
         )
         f.write(
-            "HETATM    1  C    ALK    1       "
+            "HETOM    1  C    ALK    1       "
```
Comment (sobolevnrm) [inline apbs/test/test_chemistry/test_atom_list.py:19]: HETATM

File: apbs/test/test_chemistry/test_atom_list.py [hunk 4]
```diff
@@ -1,76 +1,73 @@
...
             "2.000   5.000   8.000 0.000 2.000\n"
         )
         f.write(
-            "HETATM    1  C    ALK    1       "
+            "HETOM    1  C    ALK    1       "
```
Comment (sobolevnrm) [inline apbs/test/test_chemistry/test_atom_list.py:23]: HETATM

File: apbs/test/test_pqr/test_pqrreader.py [hunk 5]
```diff
@@ -9,13 +9,16 @@ def test_ctor(self):
         sample = r"""
 ATOM   5226  HD1 TYR   337     -24.642  -2.718  30.187  0.115 1.358
 ATOM      7  CD   LYS D   1      44.946 37.289  9.712    -0.0608  1.9080
-ATOM     39 O3PB ADP     1     -16.362  -6.763  26.980 -0.900 1.700
+REMARK This is just a comment hiding in the data
+HETOM     39 O3PB ADP     1     -16.362  -6.763  26.980 -0.900 1.700
```
Comment (sobolevnrm) [inline apbs/test/test_pqr/test_pqrreader.py:13]: No -- this should be "HETATM"!

**Review by sobolevnrm — CHANGES_REQUESTED** [review]

File: apbs/chemistry/atom.py (hunk 1 above)
Comment (intendo) [reply apbs/chemistry/atom.py:23]: Version 1.7, https://www.sphinx-doc.org/en/1.7/domains.html?highlight=send_message, search the page for `send_message`.

Comment (intendo) [reply apbs/chemistry/atom.py:23]: I can't find anywhere that states what version of Sphinx is supported in RTD.

File: apbs/test/test_chemistry/test_atom_list.py (hunk 2 above)
Comment (intendo) [reply apbs/test/test_chemistry/test_atom_list.py]: Fixed

File: apbs/test/test_chemistry/test_atom_list.py (hunk 4 above)
Comment (intendo) [reply apbs/test/test_chemistry/test_atom_list.py]: Fixed

File: apbs/test/test_pqr/test_pqrreader.py (hunk 5 above)
Comment (intendo) [reply apbs/test/test_pqr/test_pqrreader.py]: Fixed

File: apbs/test/test_chemistry/test_atom_list.py (hunk 3 above)
Comment (intendo) [reply apbs/test/test_chemistry/test_atom_list.py]: Fixed

**Review by sobolevnrm — APPROVED** [review]

**Review by ashermancinelli — APPROVED** [review]
LGTM

**Pull request merged**